knowledge_base/serverless_job/.gitignore

```diff
@@ -0,0 +1 @@
+.databricks
```

Comment:
We should probably have the same lines as https://github.com/databricks/bundle-examples/blob/main/knowledge_base/python_wheel_poetry/.gitignore here?